source/views/sis/student-work/detail.js

```diff
@@ -0,0 +1,206 @@
+// @flow
+import React from 'react'
+import {Text, ScrollView, StyleSheet} from 'react-native'
+import {fastGetTrimmedText} from '../../../lib/html'
+import {Cell, Section, TableView} from 'react-native-tableview-simple'
+import moment from 'moment'
+import openUrl from '../../components/open-url'
+import Communications from 'react-native-communications'
+import * as c from '../../components/colors'
+import type {JobType} from './types'
+
+const styles = StyleSheet.create({
+  selectable: {
+    paddingVertical: 10,
+  },
+  lastUpdated: {
+    paddingBottom: 20,
+  },
+  footer: {
+    fontSize: 10,
+    color: c.iosDisabledText,
+    textAlign: 'center',
+  },
+})
+
+function renderTitle(title: string, category: string) {
+  const trimmedTitle = fastGetTrimmedText(title)
+  const trimmedDetail = fastGetTrimmedText(category)
+  return trimmedTitle || trimmedDetail
+    ? <Section header="JOB">
+        <Cell
+          cellStyle="Subtitle"
+          title={trimmedTitle}
+          detail={trimmedDetail}
+        />
+      </Section>
+    : null
+}
+
+function renderContact(
+  title: string,
+  office: string,
+  contact: string,
+  email: string,
+) {
+  const trimmedOffice = fastGetTrimmedText(office)
+  const trimmedContact =
+    fastGetTrimmedText(contact) || fastGetTrimmedText(email)
```

Comment:
You don't need the full fastGet function for the email. The email won't have HTML in it.